include/experimental/__p1673_bits/blas3_matrix_product.hpp

```diff
@@ -537,9 +537,577 @@ void matrix_product(
   matrix_product(A, B, E, C);
 }
 
+// Overwriting triangular matrix-matrix product
+
+template<class ElementType_A,
+         extents<>::size_type numRows_A,
+         extents<>::size_type numCols_A,
+         class Layout_A,
+         class Accessor_A,
+         class Triangle,
+         class ElementType_B,
+         extents<>::size_type numRows_B,
+         extents<>::size_type numCols_B,
+         class Layout_B,
+         class Accessor_B,
+         class ElementType_C,
+         extents<>::size_type numRows_C,
+         extents<>::size_type numCols_C,
+         class Layout_C,
+         class Accessor_C>
+void triangular_matrix_left_product(
+  std::experimental::mdspan<ElementType_A, std::experimental::extents<numRows_A, numCols_A>, Layout_A, Accessor_A> A,
+  Triangle t,
```

Comment:
The `triangular_matrix_{left,right}_product` functions also take a `DiagonalStorage` template parameter and function parameter.  Please see e.g., Section **[linalg.algs.blas3.trmm.ov.left]**.   The two valid `DiagonalStorage` types are `implicit_unit_diagonal_t` and `explicit_diagonal_t`.

Reply:
Done!